### PR TITLE
Nazar/feature/validation_number_twillio

### PR DESCRIPTION
**Pull request by Nazar-kob**
Two new endpoints for phone number validation were created, the customer model was also updated, tests were added and the Twilio service was connected, but not completed yet

### CODE REVIEW

**Review by Taraslysak — APPROVED** [review]

File: backend/app/router/customer.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+)
+def create_check_customer(data: s.CreateCustomer, db: Session = Depends(get_db)):
+
+    log(log.INFO, "Create_check_customer")
+
+    is_number_valid(data.phone_number)
+
+    customer = (
+        db.query(m.Customer)
+        .filter(m.Customer.phone_number.in_([data.phone_number]))
```
Comment (Taraslysak) [inline backend/app/router/customer.py:28]: Why ```in```, but not equals?

File: backend/app/router/customer.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            )
+            raise HTTPException(
+                status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+                detail="Phone number is not valid",
+            )
+    return customer
+
+
+@router.post(
+    "/valid",
```
Comment (Taraslysak) [inline backend/app/router/customer.py:58]: I'd rather use ```validate```

**Review by Taraslysak — CHANGES_REQUESTED** [review]

File: backend/app/router/customer.py (hunk 1 above)
Comment (Nazar-kob) [reply backend/app/router/customer.py:28]: I'm thinking of the scenario that the number may be in two formats
I mean one - 380502221085
two - 0502221085

File: backend/app/router/customer.py (hunk 2 above)
Comment (Nazar-kob) [reply backend/app/router/customer.py:58]: Thanks, I corrected

File: backend/app/model/utils.py [hunk 3]
```diff
@@ -1,5 +1,10 @@
 import uuid
+from random import randint
 
 
 def gen_unique_uid() -> str:
     return str(uuid.uuid4())
+
+
+def gen_confirm_code() -> str:
+    return "".join([str(randint(0, 9)) for _ in range(6)])
```
Comment (kipod) [inline backend/app/model/utils.py:10]: here I suggest excluding value if it began from "0"

Comment (kipod) [reply backend/app/model/utils.py:10]:
```python
def gen_confirm_code() -> str:
    value="0"
    while value[0] == "0":
        value = "".join([str(randint(0, 9)) for _ in range(6)])
    return value
```

File: backend/app/router/customer.py (hunk 1 above)
Comment (kipod) [reply backend/app/router/customer.py:28]: Agree with Taras ---> need compare full number value

**Review by kipod — COMMENTED** [review]
Good job!!

File: backend/app/config.py [hunk 4]
```diff
@@ -12,6 +12,10 @@ class Settings(BaseSettings):
     ADMIN_USER: str = "admin"
     ADMIN_PASS: str = "admin"
     ADMIN_EMAIL: EmailStr = "admin@admin.com"
+    AUTH_TOKEN_TWILIO: str = ""
+    ACCOUNT_SID_TWILIO: str = ""
+    MESSAGE_SERVICE_SID: str = ""
+    COUNTRY_CODE: str = "IL"
```
[the remark this answers is not in the document]
Comment (Nazar-kob) [reply backend/app/config.py:18]: I got you

File: backend/app/model/utils.py (hunk 3 above)
Comment (Nazar-kob) [reply backend/app/model/utils.py:10]: I agree with you

**Pull request merged**